src/interface/src/app/services/plan.service.ts

```diff
@@ -727,4 +727,15 @@ export class PlanService {
         });
     }
   }
+
+  downloadCsvData(scenarioId: string): Observable<any> {
+    return this.http.get(
+      BackendConstants.END_POINT +
+        `/planning/get_scenario_download_by_id?id=${scenarioId}`,
```

Comment:
Should I change the backend to work without a queryparam? I didn't realize this was unusual for the code base.